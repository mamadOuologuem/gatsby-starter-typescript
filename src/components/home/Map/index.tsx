import React from 'react';
import {
  Typography,
  Container,
  withStyles,
  WithStyles,
  createStyles,
} from '@material-ui/core';

import tree from '../../../assets/tree.svg';

function Map({ classes }: MapProps) {
  return (
    <div className={classes.root}>
      <Container fixed>
        <Typography variant="h1">Gatsby typescript starter</Typography>
      </Container>
    </div>
  );
}

const styles = () =>
  createStyles({
    root: {
      backgroundImage: `url(${tree})`,
      backgroundSize: 400,
      backgroundRepeat: 'no-repeat',
      backgroundPosition: 'bottom right',
      width: '100vw',
      height: 'calc(100vh - 48px)',
    },
  });

export interface MapProps extends WithStyles<typeof styles> {}

export default withStyles(styles)(Map);
