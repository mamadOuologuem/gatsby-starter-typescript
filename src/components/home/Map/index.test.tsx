import React from 'react';
import { render } from '@testing-library/react';
import Map from '.';

test('renders as expected', () => {
  const { container } = render(<Map />);
  expect(container).toMatchSnapshot();
});
