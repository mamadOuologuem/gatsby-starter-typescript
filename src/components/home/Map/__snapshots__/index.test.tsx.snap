// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div>
  <div
    class="Map-root-1"
  >
    <div
      class="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
    >
      <mock-typography
        variant="h1"
      >
        Gatsby typescript starter
      </mock-typography>
    </div>
  </div>
</div>
`;
